speed-dreams/source-2.2.3/src/modules/userinterface/legacymenu/legacymenu.h

```diff
@@ -138,6 +138,10 @@ class LEGACYMENU_API LegacyMenu : public GfModule, public IUserInterface
 	//! Accessor to the sound engine.
 	ISoundEngine* soundEngine();
 
+	//! Determining the exitprotocol
+    void onAcceptExit();
+    void onAcceptRestart();
```

Comment:
anything declared in the header should be defined in the corresponding cpp file, thus this declaration should be moved to the confimationmenu.h file and exposed with 'extern' keyword.